_usage/mvu.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: Performing a Major Version Upgrade
+nav_order: 9
+---
+
+## Performing a major version upgrade
+
+Babelfish supports major version upgrade (MVU) from implementations deployed with a Babelfish PostgreSQL server version 13.6 or later. Currently the Babelfish PostgreSQL server only supports an upgrade path using the `pg_upgrade` module. During the upgrade, you will need to:
+
+- Perform a major version upgrade of the PostgreSQL server.
+- Update the Babelfish extensions to the target version.
+- Confirm that the Babelfish specific GUCs are the same in the source and target versions.
+
+Babelfish supports a major version upgrade on Babelfish PostgreSQL servers version 13.6 or later. You can use the following release as an upgrade source (the server from which you'll upgrade):
+
+- Babelfish 1.3.1, built with PostgreSQL 13.7
```

Comment:
As I understood earlier, v.1.2.2 i(=PG 1.6.4) can also be upgraded to 2.2.0.

Reply:
The OSS project has not released version 1.2.2 - I don't know yet if we have a plan to...

Reply:
we have plan to release both 1.2.2 and 1.4.0

Reply:
I should probably update the list of versions that support MVU as the versions become available...